content/docs/pipelines/sdk/static-type-checking.md

```diff
@@ -0,0 +1,144 @@
++++
+title = "DSL Static Type Checking"
+description = "Statically check the component I/O types"
+weight = 6
++++
+
+## Overview
+A pipeline is a workflow consisting of [components](/docs/pipelines/sdk/build-component#overview-of-pipelines-and-components) and each
+component contains inputs and outputs. The DSL compiler supports static type checking to ensure the type consistency among the component
+I/Os within the same pipeline. Static type checking helps users to identify component I/O inconsistencies without running the pipeline. 
```

Comment:
s/users/you

Here and throughout the doc. It's better to address the reader directly, so they know who we're talking about. "Users" could refer to the end users of their product.

Reply:
Good point.